src/core/public/injected_metadata/injected_metadata_service.test.ts

```diff
@@ -17,6 +17,7 @@
  * under the License.
  */
 
+import { PluginManifest } from '../../types';
```

Comment:
I'd like to get rid of this root `types` directory in favor of people importing types from the root of either `public` or `server`. In this case, since the server is exposing the plugin manifest for other systems (e.g. public) to consume, I would expect it to be exported from the root of server and them imported in public.

Reply:
Makes sense. I was following this types directory pattern, but I'll move it since we're going to drop it all together soon.